generators/client/templates/angular/src/test/javascript/spec/app/login/login.component.spec.ts.ejs

```diff
@@ -0,0 +1,130 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+jest.mock('@angular/router');
+jest.mock('app/core/auth/account.service');
+jest.mock('app/core/login/login.service');
+
+import { ComponentFixture, TestBed, waitForAsync, inject, fakeAsync, tick } from '@angular/core/testing';
+import { FormBuilder } from '@angular/forms';
+import { of } from 'rxjs';
+import { Router } from '@angular/router';
+
+import { LoginService } from 'app/core/login/login.service';
+import { LoginComponent } from 'app/login/login.component';
+import { AccountService } from 'app/core/auth/account.service';
+
+describe('Component Tests', () => {
+  describe('LoginComponent', () => {
+    let comp: LoginComponent;
+    let fixture: ComponentFixture<LoginComponent>;
+    let mockRouter: Router;
+    let mockAccountService: AccountService;
+    let mockLoginService: LoginService;
+
+    beforeEach(
+      waitForAsync(() => {
+        TestBed.configureTestingModule({
+          declarations: [LoginComponent],
+          providers: [
+            FormBuilder,
+            AccountService,
+            Router,
+            {
+                provide: LoginService,
+                useValue: {
+                    login: jest.fn(() => of({}))
+                }
+            }
+          ],
+        })
+          .overrideTemplate(LoginComponent, '')
+          .compileComponents();
+      })
+    );
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(LoginComponent);
+      comp = fixture.componentInstance;
+      mockRouter = TestBed.inject(Router);
+      mockLoginService = TestBed.inject(LoginService);
+      mockAccountService = TestBed.inject(AccountService);
+    });
+
+    it('Should call accountService.identity on Init', () => {
+      // GIVEN
+      mockAccountService.identity = jest.fn(() => of(null));
+      mockAccountService.getAuthenticationState = jest.fn(() => of(null));
+
+      // WHEN
+      comp.ngOnInit();
+      mockAccountService.identity().subscribe();
```

Comment:
This is called in `comp.ngOnInint();` and this test should test that, so remove this line.